source/encodingFormat-terms.ttl

```diff
@@ -1,63 +1,111 @@
+@prefix : <https://id.kb.se/vocab/> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
 @prefix owl: <http://www.w3.org/2002/07/owl#> .
 @prefix sdo: <http://schema.org/> .
 @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
-@prefix : <https://id.kb.se/vocab/> .
 @prefix iana: <https://www.iana.org/assignments/media-types/> .
-@base <https://id.kb.se/term/imt/> .
+@prefix lcmenc: <https://id.loc.gov/vocabulary/mencformat/> .
+@base <https://id.kb.se/term/encodingFormat/> .
 
 
 <application/json> a :EncodingFormat ;
-    skos:notation "application/json" ;
+    rdfs:label "application/json"@en, "applikation/json"@sv;
```

Comment:
```suggestion
    rdfs:label "JSON"
```

Reply:
Btw, I am not sure we have added lang tags for abbreviations previously... Maybe we should? :)

Reply:
Ah, there I got them switched up, well spotted! It was not intended to add lang tags on label :) Will push the corrections

Reply:
Now that labels are abbreviations, you are right, seems that the lang tags are not necessary anymore. I will remove them, maybe we should not start with that in this very moment.